### ISSUE TITLE
Whether you used dialate concolution ?

### ISSUE THREAD

**Issue by mehameha998** [otenim/GLCIC-PyTorch#7]
Hi, @otenim. Thanks for your kindly project. 
I found your implementation is a little differerent from paper content. For instance, they mentioned dialate convolution in paper, but it seems disappear in your project. I wonder how about the performance without dialate convolution?

**Comment by otenim** [comment]
@mehameha998 

Thanks for your comment.
I admit that I totally forgot to set _dilation_  argument of nn.Conv2D !!
I'll soon create another branch and do experiments on the new architecture.
Please wait little bit more until the experiments finish :)

**Comment by otenim** [comment]
Hi, @mehameha998 

I'm sorry if my codes made you confused.

Please see the following code which is at line 35 of models.py.
`self.conv7 = nn.Conv2d(256, 256, kernel_size=3, stride=1, dilation=2, padding=2)`
As you can see in the above code, I set the dilation number by setting the _dilation_ argument of nn.Conv2d.
I'm always happy to help you if you have other questions.

**Comment by mehameha998** [comment]
opzz... sorry for my careless reading. T_T Thanks you so much!
